docs/en/stack/ml/dataframes.asciidoc

```diff
@@ -0,0 +1,57 @@
+[[ml-dataframes]]
+=== {dataframes-cap}
+
+{dataframes-cap} feature is available in 7.2 and later.
+
+A _{dataframe}_ is a transformation of a dataset by certain rules defined during
+the creation of the {dataframe}. You can think of it like a spreadsheet or a 
+data table that makes your data ready to be analyzed and organized.
+
+{es} datasets consist of individual documents that have fields and
+values in each field. This architecture makes search easy but on the other hand, 
+makes it hard to run analyses that require reorganized or summarized fields of 
+the dataset. {ml-cap} analyses need clean and transformed data and that is the 
+point where {dataframes} come into play.
+
+To transform the data into a {dataframe}, you need to define a _pivot_. During
```

Comment:
"you need to define a _pivot_" : I think this can be misunderstood. _pivot_ is 1 function of a data frame, it's the only one right now, but there are plans to add more in upcoming versions. Not sure how to phrase that.

Reply:
Okay, I'll think about it and rephrase this sentence.

Reply:
I rephrased the sentence to clarify that pivot is a function of a data frame.